staker-rewards/test/data/test-blockchain.json

```diff
@@ -241,6 +197,81 @@
+          {
+            "name": "BondCreated",
+            "blockNumber": 1000,
+            "returnValues": {
+              "referenceID": "0x62f68c710615F2c86c2d224A1101112076FfcB7E",
+              "operator": "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181",
+              "sortitionPool": "0xA3748633c6786e1842b5cC44fa43db1ecC710501",
+              "amount": "20000000000000000000"
+            }
+          },
+          {
+            "name": "BondCreated",
+            "blockNumber": 1000,
+            "returnValues": {
+              "referenceID": "0x72f68c710615F2c86c2d224A1101112076FfcB7E",
+              "operator": "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181",
+              "sortitionPool": "0xA3748633c6786e1842b5cC44fa43db1ecC710501",
+              "amount": "5000000000000000000"
+            }
+          },
+          {
+            "name": "BondCreated",
+            "blockNumber": 1000,
+            "returnValues": {
+              "referenceID": "0x63f68c710615F2c86c2d224A1101112076FfcB7E",
+              "operator": "0x88Ed51f84c21Ae246c23137D090cdF441009D916",
+              "sortitionPool": "0xA3748633c6786e1842b5cC44fa43db1ecC710501",
+              "amount": "15000000000000000000"
+            }
+          },
+          {
+            "name": "BondCreated",
+            "blockNumber": 1000,
+            "returnValues": {
+              "referenceID": "0xEF833365aF1c921ED60491839F2dDCbc3cCE3630",
+              "operator": "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181",
+              "sortitionPool": "0xA3748633c6786e1842b5cC44fa43db1ecC710501",
+              "amount": "35000000000000000000"
+            }
+          },
+          {
+            "name": "BondReleased",
+            "blockNumber": 1000,
+            "returnValues": {
+              "referenceID": "0x62f68c710615F2c86c2d224A1101112076FfcB7E",
+              "operator": "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181"
+            }
+          },
+          {
+            "name": "BondSeized",
+            "blockNumber": 1000,
+            "returnValues": {
+              "referenceID": "0xEF833365aF1c921ED60491839F2dDCbc3cCE3630",
+              "operator": "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181",
+              "amount": "15000000000000000000"
+            }
+          },
+          {
+            "name": "BondReleased",
+            "blockNumber": 2000,
+            "returnValues": {
+              "referenceID": "0x72f68c710615F2c86c2d224A1101112076FfcB7E",
+              "operator": "0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181"
+            }
+          },
+          {
+            "name": "BondCreated",
+            "blockNumber": 2000,
+            "returnValues": {
+              "referenceID": "0x64f68c710615F2c86c2d224A1101112076FfcB7E",
+              "operator": "0x88Ed51f84c21Ae246c23137D090cdF441009D916",
+              "sortitionPool": "0xA3748633c6786e1842b5cC44fa43db1ecC710501",
+              "amount": "10000000000000000000"
+            }
```

Comment:
`0x88Ed51f84c21Ae246c23137D090cdF441009D916` is undelegating so they could not have `BondCreated` at the end block of the interval. If we were considering only undelegating operators, we could take ETH_locked as ETH_bonded at T_end. But we have sortition pool deauthorization as well, and this operation can be reverted by authorizing the pool again, that's why we need the minimum of ETH_bonded at the start block and at the end block. I'd rather have `BondCreated` event emitted for `0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181` at block 2000 because this could happen in a real world.

Reply:
This is just for testing purposes to make sure the logic works well. However, I'll make it more "real-world".

Reply:
https://github.com/keep-network/keep-ecdsa/pull/828/commits/17a9bc7a429af318e1d95c1f573658dba1eac732

Reply:
Should we have `BondCreated` emitted for `0x88Ed51f84c21Ae246c23137D090cdF441009D916` so that we can assert `min()` was taken into account? Or is it already covered somewhere and I can't find it?

Reply:
It's emitted in line `222`.

Reply:
Actually, the `BondReleased` at line 258 emitted at block 2000 does the min testing for `0xF1De9490Bf7298b5F350cE74332Ad7cf8d5cB181` with unauthorized pool:
Bond 20 ETH created at 1000
Bond 5 ETH created at 1000
Bond 35 ETH created at 1000
Bond 20 ETH released at 1000
Bond 15 ETH seized at 1000
Bond 5 ETH released at 2000

At block 1000, they have 20+5+35-20-15 = 25
At block 2000, they have 25 - 5 = 20
min(25,20)=20
... and this is what is asserted in the test.

All good.